src/aws/lambda-encode/src/encodeVideo.js

```diff
@@ -35,26 +35,36 @@ module.exports = async (objectKey, sourceBucket) => {
     FramerateDenominator: 1001,
     FramerateNumerator: 30000
   }
-  let videoSizes = [144, 240, 480, 720, 1080];
-  let audioBirates = [64000, 96000, 128000, 160000, 192000];
+  let videoSizesDefinition = videoSizes = [144, 240, 480, 720, 1080];
+  let audioBitratesDefinition = audioBitrates = [64000, 96000, 128000, 160000, 192000];
```

Comment:
so these could be const now?

Reply:
it can but I will have to split the variable definition

Reply:
Done